.changeset/@accounter_client-1704-dependencies.md

```
@@ -0,0 +1,9 @@
+---
+"@accounter/client": patch
+---
+dependencies updates:
+  - Updated dependency [`@radix-ui/react-checkbox@1.1.3` ↗︎](https://www.npmjs.com/package/@radix-ui/react-checkbox/v/1.1.3) (from `^1.1.3`, in `dependencies`)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)